lib/verdict/storage/base_storage.rb

```diff
@@ -43,7 +45,7 @@ def store_start_timestamp(experiment, timestamp)
         set(experiment.handle.to_s, 'started_at', timestamp.utc.strftime('%FT%TZ'))
       end
 
-
```

Comment:
Why are these protected instead of private? Are they used anywhere where they are called explicitly? e.g. `storage.get`?